10up/autoshare-for-twitter
# Feature: Add Syndication Links support

### Description of the Change


Adds support for Syndication Links with autoshare. Automatically adds the links to Twitter/X to Syndication Links meta.


Closes #79 

### How to test the Change

1. Start the included WP environment and login
2. Setup the Autoshare for Twitter plugin as normal
3. Make sure Syndication Links plugin is active
4. Add a new post with data, publsih
5. Share the post via the Autoshare plugin as normal in classic or block editor mode
6. When clicking the autoshare button in the panel to share the tweet, the link for the tweet should be automatically added to the syndication links
7. Reload the page, the link should still exist under the Syndication Links meta panel
8. Repeat steps 4 - 6, click "update" to  verify the tweet link in the Syndication Links persists

Notes:
- I didn't test with multiple accounts but it should be supported since the addition will loop through all tweet URLs. This should be tested as well.
- The addition shouldn't throw any errors when the Syndication Links plugin is not activated or available

### Changelog Entry

> Added - Support for adding Twitter/X to Syndication Links automatically
> Added - Syndication Links plugin to the WP environment config

### Credits

Props @Firestorm980 


### Checklist:
